src/Tickets/Commerce/Gateways/Stripe/Hooks.php

```diff
@@ -92,6 +99,10 @@ public function process_async_stripe_webhook( int $order_id ): void {
 			return;
 		}
 
+		if ( time() < $order->on_checkout_hold ) {
+			return;
+		}
+
```

Comment:
This should not be here. Why wait forcefully for the timeout to expire if the checkout_completed has been fired already ?